WORKSPACE

```diff
@@ -64,7 +64,7 @@ versions.check(
     maximum_bazel_version = "1.0.0",
 )  # fails if not within range
 
-load("@io_bazel_rules_go//go:deps.bzl", "go_rules_dependencies", "go_register_toolchains")
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
```

Comment:
Why the change in ordering here? I'm not sure if the ordering is significant, but the previous ordering matches the setup instructions for rules_go: https://github.com/bazelbuild/rules_go#setup

Reply:
I think it’s just reordered because of the vscode bazel extention